EIPS/eip-1682.md

```diff
@@ -32,13 +32,7 @@ upper bound on the size of the Ethereum state.
 Adding these new rules changes fundamental guarantees of the system and requires a hard
 fork. Users of Ethereum already pay for the creation and modification of accounts and
 their storage entries. Under the rules introduced in this EIP, users must also pay to keep
-accounts accessible. A similar rent scheme was proposed in [EIP-103] but rejected
```

Comment:
These very early EIPs were handled differently. They had some random number picked, and were kept in an issue with a different number.

I think removing this on the bases that it is an "invalid link" is not good enough.